LeagueOfLegendApi/lol/src/App.js

```diff
@@ -0,0 +1,103 @@
+import { useState } from "react";
+import "./App.css";
+import axios from "axios";
+import Header from "./Components/Header";
+import Tier from "./Components/Tier";
+
+function App() {
+  const [summonerName, setSummonerName] = useState("");
+  const [summonerNameData, setSummonerNameData] = useState({});
+  const [summonerSoloLeagueData, setSummonerSoloLeagueData] = useState([]);
+  const [summonerFlexLeagueData, setSummonerFlexLeagueData] = useState([]);
+  const ApiKey = "RGAPI-2e0de1b6-bda0-4c5d-84f5-19173b23ed63";
+  function searchForSummoner(event) {
+    let NameApiCallString =
+      "https://kr.api.riotgames.com/lol/summoner/v4/summoners/by-name/" +
+      summonerName +
+      "?api_key=" +
+      ApiKey;
+
+    //name 보냄 -> id, puuid 받음
+    axios.get(NameApiCallString).then(function (nameData) {
+      setSummonerNameData(nameData.data);
+      const summonerId = nameData.data.id;
+      const summonerPuuid = nameData.data.puuid;
+      let LeagueApiCallString =
+        "https://kr.api.riotgames.com/lol/league/v4/entries/by-summoner/" +
+        summonerId +
+        "?api_key=" +
+        ApiKey;
+      // id 보냄 -> leaguedata 받음
+      axios.get(LeagueApiCallString).then(function (leagueData) {
+        setSummonerSoloLeagueData(leagueData.data[0]);
+        setSummonerFlexLeagueData(leagueData.data[2]);
```

Comment:
```suggestion
        const summonerSolo = leagueData.data[0];
        const summonerFlex = leagueData.data[2];
        
        setSummonerSoloLeagueData(summonerSolo);
        setSummonerFlexLeagueData(summonerFlex);
```

leagueData.data[0] leagueData.data[2]를 보고 무슨 값을 의미 하는지 알기 힘든 거 같아요.
위처럼 표현하는 건 어떨까요?